File: app/src/main/java/com/example/allianceapp/ui/cafedra/CafedraViewModel.java
package com.example.allianceapp.ui.cafedra;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class CafedraViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public CafedraViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Кафедра");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/example/allianceapp/ui/cafedra/CafedraViewModel.kt
package com.example.allianceapp.ui.cafedra

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class CafedraViewModel : ViewModel() {
    private val mText: MutableLiveData<String>
    val text: LiveData<String>
        get() = mText

    init {
        mText = MutableLiveData()
        mText.value = "Кафедра"
    }
}